.github/workflows/check-pr-title.yaml

```diff
@@ -36,11 +36,41 @@ jobs:
       - name: Check PR title length
         if: github.event.pull_request.user.login != 'dependabot[bot]'
         env:
-          LENGTH: '70'
+          LENGTH: "70"
         run: |
           TITLE_LENGTH="$(echo -nE "$PR_TITLE" | wc --chars)"
           if [ "$TITLE_LENGTH" -gt "$LENGTH" ]
           then
             echo "::error::Please shorten the PR title from $TITLE_LENGTH to at most $LENGTH characters."
             exit 1
           fi
+
+  check-description:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+      - name: Check PR description
+        env:
+          PR_DESCRIPTION: ${{ github.event.pull_request.body }}
+        run: |
```

Comment:
Would it make sense to also verify that there is no more occurrence of "change me!" in the description ?